guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,360 @@
+
+We will first look at Labels and Contacts, then delve into how they impact/improve other features.
+
+
+## Brief explanation
+
+A silent payment transaction happens in 4 broad steps:
+- The receiver shares/publishes a static payment address
+- The sender obtains & uses it to derive a unique on-chain address
+- The sender broadcasts a transaction that pays this derived address
+- The receiver identifies the payment as theirs by identifying that they control this address
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work@2x.png"
+   mobile = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work.png"
+   mobileRetina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work@2x.png"
+   alt-text = "An image that illustrates how silent payments work in 4 steps."
+   caption = "."
+   width = 1024
+   height = 522
+   layout = "full-width"
+%}
+
+With this new model of transacting, BIP-352 introduces new primitives and concepts for users and developers. Below is a brief summary of them:
+
+| Current terminology        | With silent payments          | Benefit |
+|:-------------|:------------------|:------|
+| Single-use on-chain address           | Static address | Reusable  |
+| Extended private key (xprv) | Spend key   |   |
+| Extended public key (xpub)           | Scan key      | Addresses cannot be deterministically derived   |
+| **Address-based model**           | **Contact-based model** | **More intuitive and familiar**  |
+
+{% include image-gallery.html pages = page.images_create %}
+
+
+## Contacts & labels
+
+### Labels
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/adding-labels-to-static-address.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/adding-labels-to-static-address@2x.png"
+   modalImage = "/assets/images/guide/how-it-works/wallet-privacy/network-settings-big.png"
+   alt-text = "An application screen showing labels being added to a static address."
+   caption = "Adding labels to a static address before posting or sharing with others."
+   width = 250
+   height = 542
+   layout = "float-right-desktop -background -shadow"
+%}
+
+BIP-352 allows users to customize their silent payment addresses on a case-by-case basis by adding labels. Users may add labels to customise their static addresses before sharing or posting them. When a payment is received, these labels can be detected and used to identify the static address used to make the payment without it being obvious on-chain.
```

Comment:
Could you please verify that users can add more than 1 label? My reading from the BIP is that it's only 1, and that basically a number is added to the address, and that the wallet has match that number with some human-readable string.

Reply:
I think it's technically just one label but you can create a whole scheme of nested labels off-protocol.

Reply:
Confirmed.